### PR TITLE
[WFLY-13137] Use ee.maven.groupId in dependency management where ee.m…

### PR DESCRIPTION
**Pull request by bstansberry**
…aven.version is used

Follow-up work on https://issues.redhat.com/browse/WFLY-13137.  Use ee.maven.groupId in dependency management, and then the MP-related artifacts should use it in their pom's dependency sectioins.

### CODE REVIEW

File: microprofile/fault-tolerance-smallrye/executor/pom.xml [hunk 1]
```diff
@@ -25,12 +25,13 @@
 
     <parent>
         <groupId>org.wildfly</groupId>
-        <artifactId>wildfly-microprofile-fault-tolerance-smallrye</artifactId>
+        <artifactId>wildfly-parent</artifactId>
```
Comment (rhusar) [inline microprofile/fault-tolerance-smallrye/executor/pom.xml]: This is wrong, why this change? This breaks `mvn clean install -rf :wildfly-microprofile-fault-tolerance-smallrye`.

Comment (bstansberry) [reply microprofile/fault-tolerance-smallrye/executor/pom.xml]: We might be able to get rid of it.  It's a long story.

Comment (rhusar) [reply microprofile/fault-tolerance-smallrye/executor/pom.xml]: OK I trust you! Just wanted to point it out in case it was unintentional.
